qemu/tests/cfg/boot_order_check.cfg

```diff
@@ -6,6 +6,7 @@
     kill_vm = yes
     boot_menu = on
     enable_sga = yes
+    wait_for_devices_load = 2
```

Comment:
It's better to change this variable name to 'devices_load_timeout'.

Reply:
No problem,have done